### PR TITLE
Allow users to sign up for being notified

### PR DESCRIPTION
**Pull request by florianschmidt1994**
This PR includes functionality for users to sign up when
no help request is currently available in their region.
This is still somewhat WIP as I still want to add functionality
to always display this field but I would appreciate it if you
guys could maybe test the happy flow and report any obvious oversights!

### CODE REVIEW

File: src/components/FilteredList.js [hunk 1]
```diff
@@ -1,11 +1,65 @@
...
+
+  const [email, setEmail] = useState('');
+  const [signInLinkSent, setSignInLinkSent] = useState(false);
+
+  const handleClick = async () => {
+    window.localStorage.setItem('emailForSignIn', email);
+
+    try {
+      await firebaseApp.auth().sendSignInLinkToEmail(email, {
+        url: 'http://localhost:3000/#/complete-offer-help?location=' + location,
```
Comment (thepiwo) [inline src/components/FilteredList.js]: this url should probably be not linked to the environment or production

File: src/components/FilteredList.js [hunk 2]
```diff
@@ -1,11 +1,65 @@
...
+  const {location} = props;
+
+  const [email, setEmail] = useState('');
+  const [signInLinkSent, setSignInLinkSent] = useState(false);
+
+  const handleClick = async () => {
+    window.localStorage.setItem('emailForSignIn', email);
+
+    try {
+      await firebaseApp.auth().sendSignInLinkToEmail(email, {
```
Comment (thepiwo) [inline src/components/FilteredList.js]: maybe just await `.catch` to handle errors instead of try catch

File: src/components/FilteredList.js [hunk 3]
```diff
@@ -1,11 +1,65 @@
...
-export default function FilteredList () {
+const NotifyMe = (props) => {
+
+  const {location} = props;
+
+  const [email, setEmail] = useState('');
+  const [signInLinkSent, setSignInLinkSent] = useState(false);
+
+  const handleClick = async () => {
+    window.localStorage.setItem('emailForSignIn', email);
```
Comment (thepiwo) [inline src/components/FilteredList.js]: can we solve this in a more react way? (I don't know how)

Comment (florianschmidt1994) [reply src/components/FilteredList.js]: This is the recommended way by firebase

**Pull request merged**